### ISSUE TITLE
-iI Option Prevents Phishing

### ISSUE THREAD

**Issue by ConnorPatterson** [wifiphisher/wifiphisher#699]
Description: Thank you for [#692](https://github.com/wifiphisher/wifiphisher/pull/692), the command works now!  However, and I'm not sure if this is intentional or not, but using the -iI internetinterface option changes the behavior of the phishing script.  The commands I tried are found in the below output:

Script output: 
![screenshot from 2017-08-08 11-04-19](https://user-images.githubusercontent.com/26098605/29084771-5cdb106e-7c2a-11e7-9769-f06484907396.png)

Notice that in the first command, the output doesn't include a line like this: [*] Starting HTTTP/HTTPS server at ports 8080, 443 - but the second command, that omitted the -iI option, did.

I was testing connecting to the above created networks with my phone, and had mixed results.  The command with the -iI interface argument let me use the newly created FREEWIFI network like a regular, open access point, from which I could successfully browse the Internet, but it didn't prompt me with a phishing page in order to use the network - perhaps because it never created the phishing page server, as the output lacked evidence of such.  When I ran wifiphisher without the -iI option, the FREEWIFI network successfully redirected me to the phishing captive portal, however, after I submitted credentials successfully, it nevertheless was unable to give me an actual internet connection.  

Is it possible to make the tool do both: prompt the user with the phishing captive portal, and then give them an internet connection after they submit credentials?  I had thought initially that that was what the additional -iI option would accomplish, though I think I've misunderstood the tool :confused:

**Comment by anakin1028** [comment]
@ConnorPatterson this is correct. Currently when `-iI` option is used, we don't trigger the Captive Portal, one can only sniff the frames by the `internet adapter` for now.

However we may implement the function as you said trigger phishing page and in the mean time provide the internet access for the victims.

**Comment by yungtravla** [comment]
Use [BetterCap](https://github.com/evilsocket/bettercap) with your internet interface and enable local traffic parsing by setting `-L`

**Comment by yungtravla** [comment]
Spoofing might cause problems so make sure you also set `--no-spoofing` if you just wanna sniff the traffic.

**Comment by anakin1028** [comment]
@yungtravla This issue is related to #617. The problem here is not about sniffing, but to demonstrate the phishing pages to the victims and after that allow the victims to access the internet by the card provided with `-iI eth0` option. 

`BetterCap` is a good tool, but if you just want to sniff the internet card, one can just use `wireshark` for the `eth0`.

**Comment by yungtravla** [comment]
@anakin1028 I probably answered 2 questions at once then.

What OP wants to achieve can be done using Wifiphisher in combination with BetterCap (of course), and since Wifiphisher doesn't actually 'phish' anything (not even **traffic?**) when using the `-iI` option, again I point to [BetterCap](https://github.com/evilsocket/bettercap).
